test/fauxFetch.ts

```diff
@@ -55,11 +55,35 @@ export const FetchCreateDryRunError = async () => {
   });
```

Comment:
Really looking forward to being able to integrate local-tableland in our CI flow so we don't have to do these mocks.

Reply:
I'm hoping to work on that this week